ngx-fudis/projects/ngx-fudis/src/lib/components/dialog/dialog-directives.ts

```diff
@@ -94,4 +95,19 @@ export class DialogActionsDirective extends MatDialogActions {
 @Directive({
   selector: '[fudisDialogClose]',
 })
-export class DialogCloseDirective extends MatDialogClose {}
+export class DialogCloseDirective extends MatDialogClose {
+  private hostButtonElement: HTMLElement;
```

Comment:
Since this is a private member, let's add underscore for consistency `_hostButtonElement`.